ssc_interface_wrapper_ros2/src/converter_worker.cpp

```diff
@@ -0,0 +1,391 @@
+                                                                    std::bind(&Converter::curvature_feedback_cb, this, std_ph::_1));
+        throttle_feedback_sub_ = create_subscription<automotive_platform_msgs::msg::ThrottleFeedback>("as/throttle_feedback", 10,
+                                                                    std::bind(&Converter::throttle_feedback_cb, this, std_ph::_1));
+        brake_feedback_sub_ = create_subscription<automotive_platform_msgs::msg::BrakeFeedback>("as/brake_feedback", 10,
+                                                                    std::bind(&Converter::brake_feedback_cb, this, std_ph::_1));
+        gear_feedback_sub_ = create_subscription<automotive_platform_msgs::msg::GearFeedback>("as/gear_feedback", 10, 
+                                                                    std::bind(&Converter::gear_feedback_cb, this, std_ph::_1));
+        steering_wheel_sub_ = create_subscription<automotive_platform_msgs::msg::SteeringFeedback>("as/steering_feedback", 10,
+                                                                    std::bind(&Converter::steering_feedback_cb, this, std_ph::_1));
+                                                                                                                                                                                                                                                                                                                                                                                                                       
+        callback_from_ssc_feedbacks(velocity_feedback_, curvature_feedback_, throttle_feedback_, brake_feedback_, gear_feedback_, steering_feedback_);
+        callback_for_twist_update(velocity_feedback_,curvature_feedback_,steering_feedback_);
+
+        // Setup publishers
+        // To autoware 
+        vehicle_status_pub_ = create_publisher<autoware_msgs::msg::VehicleStatus>("vehicle_status",10);
+        current_twist_pub_ = create_publisher<geometry_msgs::msg::TwistStamped>("vehicle/twist",10);
+        // To SSC
+        steer_mode_pub_ = create_publisher<automotive_platform_msgs::msg::SteerMode>("as/arbitrated_steering_commands", 10);
+        speed_mode_pub_ = create_publisher<automotive_platform_msgs::msg::SpeedMode>("as/arbitrated_speed_commands",10);
+        turn_signal_pub_ = create_publisher<automotive_platform_msgs::msg::TurnSignalCommand>("as/turn_signal_command",10);
+        gear_pub_ = create_publisher<automotive_platform_msgs::msg::GearCommand>("as/gear_select", 1);
+
+
+        command_pub_timer_ = create_timer(get_clock(), std::chrono::duration<double>(config_.command_timeout_/2), 
+                                        std::bind(&Converter::publish_command, this));
+        // These are published in a timed callback
+        
+        status_pub_timer_ = create_timer(get_clock(),std::chrono::duration<double>(config_.status_pub_rate_),    
```

Comment:
A rate is not a duration. You need to do 1.0/rate

Reply:
This would apply for here as well? https://github.com/usdot-fhwa-stol/carma-ssc-interface-wrapper/blob/acf023d06742925ae44c825588071c55b0458363/ssc_interface_wrapper_ros2/src/ssc_interface_wrapper_node.cpp#L85